plugins/kotlin/idea/src/org/jetbrains/kotlin/idea/refactoring/move/moveDeclarations/MoveKotlinDeclarationsProcessor.kt

```diff
@@ -59,6 +61,11 @@ import kotlin.math.min
 interface Mover : (KtNamedDeclaration, KtElement) -> KtNamedDeclaration {
     object Default : Mover {
         override fun invoke(originalElement: KtNamedDeclaration, targetContainer: KtElement): KtNamedDeclaration {
+            fun KtObjectDeclaration.hasNoUsages(): Boolean {
```

Comment:
Please inline this function to avoid unnecessary overhead - it's not good to create a local function just for a single call - if you like you can use `run` that gives same short form with a `this` reference

Reply:
I inlined it.

As for my reasoning behind the function is that I found the if condition easier to read this way.